core/server/lib/mobiledoc/converters/mobiledoc-converter.js

```diff
@@ -88,14 +88,15 @@ class DomModifier {
 }
 
```

Comment:
The markdown-converter is not used in Ghost. Can i remove it?

https://github.com/TryGhost/Ghost/blob/1.25.3/core/server/lib/mobiledoc/converters/markdown-converter.js

@kevinansfield

Reply:
@kirrg001 it's definitely used in Ghost, it's required for the markdown card to work - https://github.com/TryGhost/Ghost/blob/1.25.3/core/server/lib/mobiledoc/cards/markdown.js#L9